lib/rules/lint-unless-helper.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const buildPlugin = require('./base');
+
+const messages = {
+  followingElseBlock: 'Invoking an `{{else}}` helper with the `unless` helper is not recommended',
+  followingElseIfBlock: 'Invoking an `{{else if}}` helper with the `unless` helper is not recommended',
```

Comment:
I think we can use the `followingElseBlock` for this one (e.g. we don't need two different messages in this case).